torch/csrc/utils/python_arg_parser.h

```diff
@@ -136,6 +137,7 @@ struct PythonArgs {
   inline at::Device deviceWithDefault(int i, const at::Device& default_device);
   inline c10::optional<at::Device> deviceOptional(int i);
   inline at::MemoryFormat toMemoryFormat(int i);
+  inline at::QScheme toQScheme(int i);
```

Comment:
can you check with @VitalyFedyunin what he's going to do for memory format here?  I believe this is already "memoryFormat" and that he's going to use an optional.

Reply:
Already using optional ;)

Reply:
is there anything I need to do here?